libs/sdk-model/src/entitlements/index.ts

```diff
@@ -0,0 +1,30 @@
+// (C) 2021-2023 GoodData Corporation
+
+/**
+ * Entitlement name
+ *
+ * @public
+ */
+export type IEntitlementsName =
+    | "Contract"
+    | "CustomTheming"
+    | "ManagedOIDC"
+    | "UiLocalization"
+    | "Tier"
+    | "UserCount"
+    | "PdfExports"
+    | "UnlimitedUsers"
+    | "UnlimitedWorkspaces"
+    | "WhiteLabeling"
+    | "WorkspaceCount";
+
+/**
+ * Entitlement descriptor
+ *
+ * @public
+ */
+export interface IEntitlementDescriptor {
+    name: IEntitlementsName;
+    value?: string;
```

Comment:
Document what is content of the value might be useful, is it possible to document it? Or it differs a lot across all the entitlement types?

Reply:
fixed, added some examples 48ced1ac556acf2c92b3f0f9df86fb070e23eae0